test/jdk/javax/management/MBeanServer/ExceptionTest.java

```diff
@@ -222,9 +221,6 @@ static void parseDebugProperties() {
         }
 
         /**
-         * Reproduces the original parsing and collection of test parameters
-         * from the DTonga JMX test suite.
-         *
```

Comment:
There are few similar methods
        static Map<String, Object> parseParameters(String args[])
in different Utils classes. 
Have you considered the removal of duplication code?
Also, the comment like 
" Reproduces the original parsing and collection of test parameters from legacy test suite"
might explain why this logic is needed. But it is up to you.

Reply:
Thanks -
For the paserParameters(arg) method, I think if I remove the reference to the mysterious legacy test suite, there is nothing left in that line worth saying.

The comment goes on to say "Collects passed args and returns them in a map..etc..." which seems worthwhile.

There probably is much further tidyup that we can do here, yes maybe the methods are the same, and maybe so may "Utils" classes is not great.  But I just wanted to remove this one point of confusion today!

Reply:
The link to legacy is a just good explanation why this method is realized in such way. Like note, "There is might be no logic in this method, just legacy."

Reply:
Yes, better to rewrite it of course, but now it becomes unclear why it was written at all.

Reply:
I will go with this change for now, maybe we can come back here!